scripts/install_linux_deps.sh

```diff
@@ -6,9 +6,12 @@ DISTRIB=$ID
 
 ARCH_DEPS="boost boost-libs expect jq autoconf shellcheck sqlite python-virtualenv"
 UBUNTU_DEPS="libboost-all-dev expect jq autoconf shellcheck sqlite3 python3-venv"
+FEDORA_DEPS="boost-devel expect jq autoconf ShellCheck sqlite python-virtualenv"
 
 if [ "${DISTRIB}" = "arch" ]; then
     pacman -S --refresh --needed --noconfirm $ARCH_DEPS
+elif [ "${DISTRIB}" = "fedora" ]; then
+    dnf -y install $FEDORA_DEPS
```

Comment:
maybe add something like "not officially supported, use fedora build on your own risk"